SourceLink.Create.CommandLine/CreateTask.cs

```diff
@@ -67,7 +67,8 @@ public static string ConvertUrl(string origin)
         {
             var urlConverters = new List<UrlConverter> {
                 GitHub.UrlConverter.Convert,
-                BitBucket.UrlConverter.Convert
+                BitBucket.UrlConverter.Convert,
+                BitBucketServer.UrlConverter.Convert
```

Comment:
I'm not sure this should be in the default list of converters. Probably revert this change and we should have a new feature that allows you to specify the converter.

Reply:
@ctaggart reverted as requested